rpc-jobs/jjb-setup.yml

```diff
@@ -1,4 +1,24 @@
 - job:
+    name: 'Merge-Trigger-JJB'
+    project-type: freestyle
```

Comment:
The trigger job I wrote was a pipeline/workflow type instead of a freestyle (https://github.com/rcbops/rpc-gating/pull/100), but I know you were having issues making this one a workflow, so I might change mine over to a freestyle too just so that we have a consistent format for trigger jobs.